source/deployment/sso-saml-adfs.md

```diff
@@ -96,6 +96,9 @@ However, if you would like to set up encryption for your SAML connection, click
 
 For Mattermost 3.4 and later, the `FirstName` and `LastName` attributes are optional.
 
+.. note::
+It is recommended that you also set up an attribute that is associated with a unique, non-changing identifier to prevent username or email collisions.
```

Comment:
```suggestion
  It is recommended that you also set up an attribute that is associated with a unique, non-changing identifier to prevent username or email collisions.
```